addons/delivery/models/delivery_carrier.py

```diff
@@ -59,6 +61,10 @@ def toggle_prod_environment(self):
         for c in self:
             c.prod_environment = not c.prod_environment
 
+    def toggle_debug(self):
+        for c in self:
+            c.debug_logging = not c.debug_logging
```

Comment:
Not sure what the access rights are, but maybe only the admin should be allowed to do that? Or this entire thing should be packaged in a debug_shippers module?